src/modules/uORB/uORBDeviceNode.cpp

```diff
@@ -70,6 +71,11 @@ uORB::DeviceNode::~DeviceNode()
 	if (_data != nullptr) {
 		delete[] _data;
 	}
+
+	if (_devname != nullptr) {
```

Comment:
This does not work: `CDev::~CDev()` is called after this, which will access the string (use-after-free).
You can add an unregister method to `CDev` and call it from here.